hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -460,21 +460,22 @@ private BitSet setMatchedPartitions(String partitionIds, CarbonTable carbonTable
       for (DataRefNode dataRefNode : dataRefNodes) {
         BlockBTreeLeafNode leafNode = (BlockBTreeLeafNode) dataRefNode;
         TableBlockInfo tableBlockInfo = leafNode.getTableBlockInfo();
+        String[] deleteDeltaFilePath = null;
         if (isIUDTable) {
           // In case IUD is not performed in this table avoid searching for
           // invalidated blocks.
           if (CarbonUtil
-              .isInvalidTableBlock(tableBlockInfo.getSegmentId(), tableBlockInfo.getFilePath(),
-                  invalidBlockVOForSegmentId, updateStatusManager)) {
+                  .isInvalidTableBlock(tableBlockInfo.getSegmentId(), tableBlockInfo.getFilePath(),
```

Comment:
`CarbonInputFormat.java` is not used now. please use `CarbonTableInputFormat`

Reply:
Do i need to revert the changes done in CarbonInputFormat.java

Reply:
you can keep here , not an issue since this class going to be removed soon. Please do changes in `CarbonTableInputFormat`

Reply:
Changes done in CarbonTableInputFormat and changes is pushed